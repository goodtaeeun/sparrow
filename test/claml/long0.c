int main() {
  int x = 9223372036854775807L;
  return 0;
}
