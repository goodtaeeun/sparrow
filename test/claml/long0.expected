/* Generated by CIL v. 1.7.4 */
/* print_CIL_Input is false */

#line 1 "long0.c"
int main(void) {
  int x;

  {
#line 2
    x = (int)-1L;
#line 3
    return (0);
  }
}
